README.adoc

```diff
@@ -131,7 +161,15 @@ APIM supports two types of authorization server:
     "oauthCacheResource": "cache-resource-name",
     "extractPayload": true,
     "checkRequiredScopes": true,
-    "requiredScopes": ["openid", "resource:read", "resource:write"]
+    "requiredScopes": ["openid", "resource:read", "resource:write"],
+    "confirmationMethodValidation" : {
+      "ignoreMissing":  false,
+      "certificateBoundThumbprint" : {
+        "enabled": false,
+        "extractCertificateFromHeader": false,
```

Comment:
For the example we could enabled these feature wdyt?

Reply:
Could be true or false, as an example it shows the default value.